test/hotspot/gtest/utilities/test_vmerror.cpp

```diff
@@ -28,6 +28,17 @@
 
 #ifdef ASSERT
 
+#ifdef _WINDOWS
+
+#include <windows.h>
+
+TEST_VM_ASSERT_MSG(vmErrorTest, fatalWithError,
+                   "fatal error: GetLastError should be 6 - actually: 6") {
+  SetLastError(6);
+  fatal("GetLastError should be 6 - actually: %d", (int)GetLastError());
```

Comment:
Why is this casting the value of GetLastError, rather than using "%u" in the format string?

Reply:
Simply because when I initially was testing this I used a call that had `%d` as the "template" - see e.g. perfMemory_windows.cpp. But for consistency with other uses in os_windows.cpp I should be using `%lu` and no cast.